features/step_definitions/find_car_steps.rb

```diff
@@ -0,0 +1,202 @@
+Given /^search page is opened$/ do
+  FindCarPage.open
+end
+
+When /^user fill in Postcode field with (\S+)$/ do |data|
+  FindCarPage.given.enter_data(data)
```

Comment:
`enter_data` method name is too generic, use something like: `enter_postcode`. Don't use word `data` too, it's too generic.